src/libs/ajax.js

```diff
@@ -442,15 +442,24 @@ const Billing = signal => ({
     return res.json()
   },
 
+  getProject: async projectName => {
+    const route = `billing/v2/${projectName}`
+    const res = await fetchRawls(route, _.merge(authOpts(), { signal, method: 'GET' }))
+    return res.json()
+  },
+
   listAccounts: async () => {
     const res = await fetchRawls('user/billingAccounts', _.merge(authOpts(), { signal }))
     return res.json()
   },
 
   createProject: async (projectName, billingAccount) => {
-    const res = await fetchRawls('billing/v2',
-      _.mergeAll([authOpts(), jsonBody({ projectName, billingAccount }), { signal, method: 'POST' }]))
-    return res
+    const res = await fetchRawls('billing/v2', _.mergeAll([
+      authOpts(),
+      jsonBody({ projectName, billingAccount }),
+      { signal, method: 'POST' }
+    ]))
```

Comment:
I don't love breaking this into 3 lines; I cold see this all on one line, but I think if we're adding linebreaks I might go for 
```js
fetchRawls(
  'path',
  _.mergeAll([])
)
```